src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -362,9 +382,8 @@ def body(self):
         self.cmd('sql elastic-pools db show-activity -g {} --server-name {} --elastic-pool-name {}'
                  .format(rg, self.sql_server_name, self.pool_name),
                  checks=[
-                     JMESPathCheck('[0].resourceGroup', rg),
-                     JMESPathCheck('[0].serverName', self.sql_server_name),
-                     JMESPathCheck('[0].currentElasticPoolName', self.pool_name)])
+                     JMESPathCheck('type(@)', 'array'),
+                     CustomCheck(self.custom_activity_checker)])
```

Comment:
@derekbekoe I thought there was a way through a regular JMESPath query to basically check each element in the collection, which essentially is all we want to do here. It just seems that the CustomCheck isn't really needed. If we can't check it with a regular JMESPath check, then I would be fine with just looping through with the assert statements.